qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -899,7 +899,18 @@ def _inject_userscripts(self):
             # @run-at (and @include/@exclude/@match) is parsed by
             # QWebEngineScript.
             new_script = QWebEngineScript()
-            new_script.setWorldId(QWebEngineScript.MainWorld)
+            try:
+                world = int(script.jsworld)
+            except ValueError:
+                try:
+                    world = [v for k, v in _JS_WORLD_MAP.items()
+                             if k.name == script.jsworld.lower()][0]
```

Comment:
You can just do `world = usertypes.JsWorld[script.jsworld.lower()]` instead (and catch `KeyError`).